### DISCUSSION TITLE
disable throttle after 30 second and reset

### DISCUSSION THREAD

**Discussion by GiacomoGaleotti** [FrSkyRC/ETHOS-Feedback-Community#4037] [Q&A] [answered]
i'm struggling to set the radio to cut throttle after 30 seconds
then i would like to be able to reset the situation with a momentary button to enable again the throttle slider

if possible, i guess i may use this function as the role to stop countdown timers
i'm still not able to get those countdown stop at zero

**Comment by GiacomoGaleotti** [comment]
No one can give some help? Please

**Comment by GiacomoGaleotti** [comment]
Super team of helper!! Thanks a lot 😣

**Comment by bsongis** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/5603504/30b2949e-0d70-44ab-a9d0-d583a9ea4860)

**Comment by bsongis** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/5603504/5f73c34b-e107-44d1-aa7e-a1e6878df590)

**Comment by bsongis** [comment, marked as answer]
1.5.9 only allows 25s, but 30s will be allowed in 1.5.10

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/5603504/62cea723-daba-48d0-b7bb-7774d04eacbc)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/5603504/3ce06fcb-a3df-4db1-9279-e5f3146e4774)

**Reply by GiacomoGaleotti** [reply]
Thanks a lot, I was trying both this ways with same problem of 25 sec, so I was wondering if there was another solution to walk around it but seem not 😜
thanks for your help and I'll wait for next release
